### PR TITLE
M3 : Merge Community Version 3.1.1

### PR DESCRIPTION
**Pull request by tejasync**
Fix failing test cases and warnings in https://github.com/acquia/mc-cs/pull/1550

### CODE REVIEW

File: Config/config.php [hunk 1]
```diff
@@ -65,6 +65,7 @@
                     'mautic.core.model.notification',
                     'mautic.lead.model.field',
                     'mautic.plugin.model.integration_entity',
+                    'mautic.lead.model.dnc',
```
Comment (shinde-rahul) [inline Config/config.php:68]: There is no need for this additional as the AbstractIntegration class has only 15 constructor arguments. https://github.com/acquia/mc-cs/blob/epic-mautic3.3.3/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L200-L216
Am I missing something?

Comment (tejasync) [reply Config/config.php:68]: Yes, you should really check MAUT-6009 branch: https://github.com/acquia/mc-cs/blob/MAUT-6009/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L237

**Review by rohitp19 — APPROVED** [review]
Approving it as code review is already done by Team.

**Review by dadarya0 — APPROVED** [review]
Approving it as code review is already done by Team.

**Pull request merged**